framework/src/modules/reward/endpoint.ts

```diff
@@ -12,6 +12,43 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
+import { ModuleEndpointContext } from '../..';
 import { BaseEndpoint } from '../base_endpoint';
+import { DefaultReward, EndpointInitArgs, GetDefaultRewardParams } from './types';
 
-export class RewardEndpoint extends BaseEndpoint {}
+export class RewardEndpoint extends BaseEndpoint {
+	private _brackets!: ReadonlyArray<bigint>;
+	private _offset!: number;
+	private _distance!: number;
+
+	public init(args: EndpointInitArgs) {
+		this._brackets = args.config.brackets;
+		this._offset = args.config.offset;
+		this._distance = args.config.distance;
+	}
+
+	public getDefaultRewardAtHeight(ctx: ModuleEndpointContext): DefaultReward {
```

Comment:
`DefaultReward` has `bigint` in the property, which is not valid JSON format. It should be converted to `string`

Reply:
done